lib/fake-xhr/index.test.js

```diff
@@ -1995,6 +1995,22 @@ describe("FakeXMLHttpRequest", function () {
                     FakeXMLHttpRequest.useFilters = false;
                 }
             });
+
+        it("correctly calls events on the fakeXhr",
+            function () {
+                try { // eslint-disable-line no-restricted-syntax
+                    FakeXMLHttpRequest.useFilters = true;
+                    var spy = sinonSpy();
+                    fakeXhr.addEventListener("abort", spy);
+                    fakeXhr.onabort = spy;
+                    fakeXhr.open("GET", "http://example.com", true);
+                    fakeXhr.send();
+                    fakeXhr.abort();
+                    assert.equals(spy.callCount, 2);
```

Comment:
It might make this test easier to read if you explain why we expect `spy.callCount` to be `2`.

Alternatively, you could also create two spies, and assign one to `.onabort` and use the other as argument to `addEventListener`.

Reply:
Which one would you prefer? 2 spies or a message on the assert?

Reply:
Sorry, I don't know where my head is this morning ....

Reply:
No worries at all. This just verifies that the initial listener that is added in the constructor for fakeXhr.onabort is called as well as any other other events added via addEventListener.

Reply:
I think that also answered the question. 2 different spies would make more sense here.

Reply:
👍 

I would prefer two listeners